frontend/src/libs/components/scroll-to-top-button/scroll-to-top-button.tsx

```diff
@@ -0,0 +1,56 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback, useEffect, useState } from '~/libs/hooks/hooks.js';
+
+import { IconButton } from '../components.js';
+import styles from './styles.module.scss';
+
+const ScrollToTop: React.FC = () => {
+  const [showButton, setShowButton] = useState(false);
+  const [scrollValue, setScrollValue] = useState(0);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > 200) {
```

Comment:
**B31**
And as there is many values related to scrolling - you can move them out to enum